apps/designer/app/designer/features/sidebar-left/panels/asset-manager/asset-info.tsx

```diff
@@ -0,0 +1,130 @@
+import {
+  Box,
+  Flex,
+  Grid,
+  IconButton,
+  Text,
+  Separator,
+  Button,
+} from "@webstudio-is/design-system";
+import type { Asset } from "@webstudio-is/prisma-client";
+import { getFormattedAspectRatio } from "./utils";
+import {
+  CloudIcon,
+  AspectRatioIcon,
+  Cross1Icon,
+  SizeIcon,
+  TrashIcon,
+} from "@webstudio-is/icons";
+import { useSubmit } from "@remix-run/react";
+import prettyBytes from "pretty-bytes";
+
+const truncatedText = {
+  position: "relative",
+  maxWidth: 80,
+
+  div: {
+    whiteSpace: "nowrap",
+    textOverflow: "ellipsis",
+    overflow: "hidden",
+
+    "&:after": {
+      content: "attr(data-extension)",
+      position: "absolute",
+      left: "100%",
+      top: 0,
+      whiteSpace: "nowrap",
+    },
+  },
+};
+
+type AssetInfoProps = Asset & {
+  onClose: () => void;
+  setIsDeleting: (isDeleting: boolean) => void;
+};
+
+export const AssetInfo = ({
+  size,
+  width,
+  height,
+  id,
+  name,
+  onClose,
+  setIsDeleting,
+}: AssetInfoProps) => {
+  const submit = useSubmit();
+  const handleDeleteAsset = () => {
+    const formData = new FormData();
+    formData.append("assetId", id);
+    formData.append("assetName", name);
+    onClose();
+    setIsDeleting(true);
+    submit(formData, { method: "delete" });
+  };
+
+  return (
+    <>
+      <Flex
+        css={{ height: 40, paddingLeft: "$3" }}
+        align="center"
+        justify="between"
+      >
+        <Text size="1" weight="700">
+          Asset Details
+        </Text>
+
+        <IconButton
+          onClick={onClose}
+          size="1"
+          css={{ marginRight: "$2" }}
+          aria-label="Close"
+        >
+          <Cross1Icon />
+        </IconButton>
+      </Flex>
+      <Separator />
+      <Box css={{ p: "$2 $3" }}>
+        <Grid columns={2} align="center">
+          <Box css={{ width: 100 }}>
+            <Box css={truncatedText}>
+              <Text size="1" data-extension={name.split(".").pop()}>
```

Comment:
could this go sideways if I have the same chats as the extension inside the name anywhere?

Reply:
This splits by . and gets the last one, the last one will always be the extension doesn't matter how many `.` you have

Reply:
I meant if the extension is inside the word but maybe I misunderstood, then ignore me

Reply:
Extension is inside the word? What?

Reply:
logo-as-png.png

Reply:
Nope, that's not an issue because it will split at the . And still get the last png